modules/composer-workflowoperation/src/main/java/org/opencastproject/workflow/handler/composer/CompositeWorkflowOperationHandler.java

```diff
@@ -345,6 +353,7 @@ private WorkflowOperationResult composite(MediaPackage src, WorkflowOperationIns
     CompositeSettings(MediaPackage mediaPackage, WorkflowOperationInstance operation)
             throws WorkflowOperationException {
       // Check which tags have been configured
+      sourceAudioName = StringUtils.trimToNull(operation.getConfiguration(SOURCE_AUDIO_NAME));
```

Comment:
Suggestion: I would recommend to default to "both" (not to null). This slightly simplifies further handling as you don't need to check for null anymore.